core/profiles/standard/config/layout.layout.home.json

```diff
@@ -6,7 +6,7 @@
     "description": null,
     "module": "standard",
     "weight": 0,
-    "storage": 4,
+    "storage": 2,
```

Comment:
This is what "flags" a config file as "modified", which triggers the "revert" option to become available. Do we want that here?

Reply:
Probably not. I probably copied that value from an existing config file without understanding why. I'll look at it closer. 

I'm not too worried about code yet, because there are so many decisions yet to be made.

Reply:
Reverted this in latested PR